packages/react/src/errorboundary.tsx

```diff
@@ -107,7 +113,8 @@ class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundarySta
         beforeCapture(scope, error, passedInComponentStack);
       }
 
-      const eventId = captureReactException(error, errorInfo, { mechanism: { handled: !!this.props.fallback } });
+      const isHandled = this.props.handled === undefined ? !!this.props.fallback : this.props.handled;
+      const eventId = captureReactException(error, errorInfo, { mechanism: { handled: isHandled } });
```

Comment:
a suggestion to make this a bit more concise and save some bytes of bundle size. I turned around the condition to reflect that the fallback is actually our fallback way of determining handled:

```suggestion
      const handled = this.props.handled != null ? this.props.handled : !!this.props.fallback;
      const eventId = captureReactException(error, errorInfo, { mechanism: { handled } });
```

(`!= null` can be used as a shorthand for `!== undefined && !== null`)

Reply:
Fixed in 11c408df6906b93ca5b94270bd0f672a659d69f2